build.gradle

```diff
@@ -322,6 +373,8 @@ rat {
     reportDir.set(file("build/reports/rat"))
 }
 
+distZip.dependsOn copyJolokia
+distTar.dependsOn copyJolokia
```

Comment:
we fixed installDist, but forgot to address distZip and distTar in CASSSIDECAR-122